chord_metadata_service/chord/ingest/phenopackets.py

```diff
@@ -236,14 +236,14 @@ def get_or_create_genomic_interpretation(gen_interp: dict) -> pm.GenomicInterpre
     gene_descriptor = _get_or_create_opt("gene_descriptor", gen_interp, get_or_create_gene_descriptor)
     variant_interpretation = _get_or_create_opt("variant_interpretation", gen_interp, get_or_create_variant_interp)
 
-    gen_obj, _ = pm.GenomicInterpretation.objects.get_or_create(
+    gen_obj, created = pm.GenomicInterpretation.objects.get_or_create(
         interpretation_status=gen_interp["interpretation_status"],
         gene_descriptor=gene_descriptor,
         variant_interpretation=variant_interpretation,
         extra_properties=gen_interp.get("extra_properties", {}),
     )
 
-    if related_obj:
+    if related_obj and created:
```

Comment:
why this change? I'm not sure what the original problem was, but if the object is re-used and the related object doesn't already have the object as an interpretration, shouldn't it still be added?

Reply:
Removed, I was testing stuff out and forgot to remove this check.